### DISCUSSION TITLE
works on Windows 7!

### DISCUSSION THREAD

**Discussion by ghost** [Godiesc/firefox-gx#70] [General]
this is a really neat theme, and i'd love to show it to friends who like Opera GX!
i noticed the readme mentions Windows 7 hasn't been tested, so i've given it a try and all looks okay!
this is with Firefox 113, by the way
![Screenshot](https://github.com/Godiesc/firefox-gx/assets/83787150/bf0717f4-58fb-4d3d-b085-ed3f07f158f9)

**Comment by Godiesc** [comment]
Thanks, with some visual issues but works fine, it seems, but with the next version the browser isn't compatible with windows 8 or lower versions, I don't know how that will work.

**Comment by almaridi** [comment]
mentions Windows 7 hasn't been tested, so i've given it a try and all looks okay!
this is with Firefox 115 Thanks
![ScreenShot_20231215082142](https://github.com/Godiesc/firefox-gx/assets/36901313/ac3f81f1-5e3e-446f-87ca-341ada01e6ee)
